misc/io.github.grangerhub.Tremulous.appdata.xml

```diff
@@ -19,16 +19,47 @@
     <p>
       Players can choose from 2 unique races, aliens and humans.
       Players on both teams are able to build working structures in-game like an
-      RTS.
+      RTS. 
+    </p>
+    <p>
+      This snap provides the Grangerhub updated client binary that
+      supports new features.
     </p>
   </description>
   <url type="homepage">https://github.com/GrangerHub/tremulous</url>
   <screenshots>
-    <screenshot type="default"><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/a.png</image></screenshot>
-    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/b.png</image></screenshot>
-    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/c.png</image></screenshot>
-    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/d.png</image></screenshot>
-    <screenshot><image>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/tremulous/e.png</image></screenshot>
+    <screenshot type="default">
+      <image type="source" width="1280" height="960">https://github.com/GrangerHub/tremulous/blob/master/misc/grangerhub-trem1.3-7d1b.jpg?raw=true</image>
```

Comment:
Just a note: I may be wrong, but I think that the correct URL should be `https://raw.githubusercontent.com/GrangerHub/tremulous/master/misc/grangerhub-trem1.3-7d1b.jpg`.

Same with the other screenshots/videos.

Reply:
Yeah, this sounds right. I used to be able to get those URLs but somehow I can't seem to get to them. But just using your example works (for existing files). I will re-confirm once I merge this PR to master.